optionals/compat_ace/stringtable.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ZEN">
+    <Package name="Compat_ACE">
+        <Key ID="STR_ZEN_Compat_ACE_HideModules">
+            <English>Hide ACE Modules</English>
+        </Key>
+        <Key ID="STR_ZEN_Compat_ACE_HideModules_Description">
+            <English>Hides ACE modules for which replacements exist. Requires display reload.</English>
```

Comment:
```suggestion
            <English>Hides ACE modules for which a replacement exists. Requires display reload.</English>
```

Reply:
I think "replacements" is correct since "modules" is plural.

Reply:
Hm, personally I agree with neilzar since `a replacement exists` is a condition applying to each module individually and not all. In my opinion it sounds more natural.

Reply:
To me, the current form suggests that multiple replacements can exist for each module.

Reply:
Changed to: "Hides duplicate modules added by ACE Zeus". Think that is the most clear description.